### PR TITLE
Added functionality for handling blurry form submissions.

### PR DESCRIPTION
**Pull request by iyves**

### CODE REVIEW

File: Appserver/Controllers/ImageUploadController.cs [hunk 1]
```diff
@@ -42,14 +44,41 @@ public async Task<IActionResult> PostImage(List<IFormFile> files, AbstractFormOb
             {
                 "image/jpeg",
                 "image/png",
+                "image/heic",
                 "application/pdf",
+                "application/octet-stream"
             };
+            // Detect blur for each image
+            double threshold = 1000.0;
```
Comment (ohyonghao) [inline Appserver/Controllers/ImageUploadController.cs]: Value too high

**Review by ohyonghao — CHANGES_REQUESTED** [review]
Boots out forms that aren't blurry

**Review by unstable-compound — DISMISSED** [review]

**Review by ohyonghao — CHANGES_REQUESTED** [review]
Regression found in PWA giving blank forms for blurry image

**Comment by iyves** [comment]
What does this mean? Any steps to reproduce? What exactly do you want changed?

**Comment by ohyonghao** [comment]
When I submit an image that is below the threshold the Appserver responds with "too blurry", but the PWA continues to the next stage with a blank form.

**Review by keatonkraiger — DISMISSED** [review]

File: TextToDoc/Images.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                    if(String.IsNullOrEmpty(contenttype))
+                    {
+                        break;
+                    }
+                    var imageFile = File.OpenRead(file);
+                    using (var ms = new MemoryStream())
+                    {
+                        imageFile.CopyTo(ms);
+                        ms.Seek(0, SeekOrigin.Begin);
+                        var blur = ImageUploadController.detect_blur(new FormFile(ms, 0, ms.Length, imageFile.Name, imageFile.Name)
```
Comment (iyves) [inline TextToDoc/Images.cs]: @keatonkraiger @ohyonghao This is the line of code that breaks the unit test, after I did a pull from develop. Should it be 'DocumentUploadController?'

Comment (ohyonghao) [reply TextToDoc/Images.cs]: Yes, that should be updated to DocumentUploadController.

**Comment by iyves** [comment]
o yea boi we in dis

**Review by ohyonghao — DISMISSED** [review]

**Review by ohyonghao — APPROVED** [review]

**Pull request merged**